decentraland/unity-explorer
# fix: fixed cursor size for mac

## What does this PR change?

This PR reduces the max size of the sprites used for the cursor, fixing the gigantic size on mac

...

## How to test the changes?



1. Launch the explorer
2. Check the cursor size

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
